prem/DivAndSpan.html

```diff
@@ -0,0 +1,111 @@
+    <title>
+        DivAndSpan
+    </title>
+    <style>
+      
+        body {
+            width: 98.5vw;
+            height: 96.5vh;
+            border: 2px solid black;
+        }
+        .main-section {
+            position: absolute;
+            width: 50%;
+            height: 50%;
+            margin: auto;
+            top: 0;
+            bottom: 0;
+            left: 0;
+            right: 0;
+        }
+
+        .head {
+            position: absolute;
+            box-sizing: border-box;
+            text-align: left;
+            width: 100%;
+            height: 12%;
+            padding: 1%;
+            border: 2px solid black;
+        }
+
+        .foot {
+            position: absolute;
+            width: 100%;
+            box-sizing: border-box;
+            padding: 1%;
+            height: 12%;
+            bottom: 0;
+            border: 2px solid black;
+        }
+
+        span {
+            font-weight: bold;
+        }
+
+        .first-div {
+            position: relative;
+            box-sizing: border-box;
+            padding: 1%;
+            margin-top: 7.5%;
+            border: 2px solid black;
+        }
+
+        .second-div {
+            position: relative;
+            height: 18%;
+            box-sizing: border-box;
+            overflow: hidden;
+            padding: 1%;
+            margin-top: 2%;
+            border: 2px solid black;
+        }
+
+        .third-div {
+            position: relative;
+            height: 18%;
+            box-sizing: border-box;
+            padding: 1%;
+            overflow-x: hidden;
+            overflow-y: scroll;
+            margin-top: 2%;
+            border: 2px solid black;
+        }
+    </style>
+</head>
+
+<body>
+    <section class="main-section">
+        <div class="head">
+            Header
+        </div>
+        <div class="first-div">
+            <span>Fit to Content - </span>The CSS box model is the foundation of layout on the Web — each
+            element is represented as a rectangular box, with the box's
+            content, padding, border, and margin built up around one another
+            like the layers of an onion.
+        </div>
+        <div class="second-div">
+            <span>Overflow Hidden - </span>The CSS box model is the foundation of layout on the Web — each
+            element is represented as a rectangular box, with the box's
+            content, padding, border, and margin built up around one another
+            like the layers of an onion.
+        </div>
+        <div class="third-div">
+            <span>Overflow Scroll - </span>The CSS box model is the foundation of layout on the Web — each
+            element is represented as a rectangular box, with the box's
+            content, padding, border, and margin built up around one another
+            like the layers of an onion.
+        </div>
+        <div class="foot">

```

Comment:
What is the need for having a div with class="foot"? Can you please find some alternative for this element?

Reply:
i will change it with suitable class name.